baybe/recommenders/pure/bayesian/base.py

```diff
@@ -92,7 +92,7 @@ def _fit(
         train_x: pd.DataFrame,
         train_y: pd.DataFrame,
     ) -> Surrogate:
-        """Train a fresh surrogate model instance for the DOE strategy.
+        """Train a fresh surrogate model instance for the DOE recommender.
```

Comment:
Unsure if we should keep strategy here, as I think this did not necessarily refer to the object, right?

Reply:
i read this as "train something for the recommender", I would remove the DOE and name it `recommender`

but the alternative phrasing that would read "does something for the optimziation strategy", however the first one imo is more precise about the code

Reply:
changed, imo its completely suffiecient to say `Train a fresh surrogate model instance.`